### ISSUE TITLE
docs: add documentation on fixed content

### ISSUE THREAD

**Issue by brandyscarney** [driftyco/ionic-site#776]
https://github.com/driftyco/ionic/issues/5987

**Comment by movrack** [comment]
The feature always exist? I can't find it.

**Comment by brandyscarney** [comment]
@movrack Yes, `ion-fixed` was converted into an attribute. See the following test for an example of it being used: https://github.com/driftyco/ionic/blob/master/src/components/nav/test/basic/app-module.ts#L390-L391

And a screenshot of that test (the fixed button and large grey box are fixed):

![localhost-8000-dist-e2e-nav-basic- windows phone](https://cloud.githubusercontent.com/assets/6577830/20014522/b99a2e7e-a28d-11e6-99a9-8abcf98a29f5.png)

**Comment by ix-xerri** [comment]
It party worked. The content stays in .scroll-content and fixed content remains empty and I am using ion-segment and the segment where I don't have google map but an ion-card does not show. The content is hidden.

**Comment by Shehab-Muhammad** [comment]
I can't use ion-fixed right now, is it possible to put content into container with class `.fixed-content` ? how ?
